src/components/CommentList.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import Commentar from './Comment';
+
+export default class CommentList extends Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components